digitalpublicgoods/flow-cameroon.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "Flow Cameroon",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://github.com/Idadelveloper/Flow/blob/main/README.md"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "Yes",
+    "isSoftwarePltIndependent": "Yes",
+    "pltIndependenceDesc": "Flow Cameroon requires Firebase and Firestore which are closed dependencies. However other open source alternatives exists such Couchbasev (https://www.couchbase.com/)"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/Idadelveloper/Flow/blob/main/README.md"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
```

Comment:
I believe this answer is not correct, since this application collects the location of water sources, and that is nonPII. @nathanbaleeta please contact the product owner to clarify. Thanks!

Reply:
Resolved in commit 0e55394